templates/static/js/src/demobox/components/LiveDemoBox/Code.tsx

```diff
@@ -61,6 +67,17 @@ const Code = ({ path }: Props) => {
               />
             </div>
           )}
+          <div className="p-code-snippet__codepen-link">
+            <CodepenPrefill
```

Comment:
Could you move this to below the iframe in LiveDemoBox.tsx to better match the design? I think it would go well right below [this line](https://github.com/canonical-web-and-design/vanilla-framework/blob/live-demo-dox/templates/static/js/src/demobox/components/LiveDemoBox/LiveDemoBox.tsx#L119). (You can wrap it in a `p-card__inner` div to give it the proper padding inside the card.)

Reply:
The design has the HTML dropdown together with the Edit on Codepen link, that's why I put it there and we also have the html, css and js data already there in the component?

Reply:
I would suggest moving the CodePen link into [this div](https://github.com/canonical-web-and-design/vanilla-framework/pull/4340/files#diff-5d1da8d8c9c5e5d9473769d55ff7f28c649eb0950245f4b405233cba2fa1cdebR121). You can call the `useCode()` hook in LiveDemoBox.tsx and pass it the `url` variable that is set there.

Reply:
well, it's done.